src/components/MultiSelect.js

```diff
@@ -73,7 +106,10 @@ const MultiSelectItem = ({
         {label}
       </Typography>
       {!disabled && (
-        <TouchableWithoutFeedback disabled={disabled} onPress={onUnselect}>
+        <TouchableWithoutFeedback
+          disabled={disabled}
+          onPress={handleItemUnSelection}
```

Comment:
@Tsudhishnair A small suggestion, we are showing the `Alert` for every delete action. There might be some cases when we don't need confirmation, so can we support another prop to disable the confirmation and delete the item from the list directly? We can set the prop as enabled by default and only when the user doesn't need confirmation then the prop can be passed and the Alert will be bypassed.

Cc: @bigbinary/neeto-rn Open for suggestions

Reply:
@injas-bigbinary @Tsudhishnair Yes agree.